e2e/cypress/support/commands.js

```diff
@@ -132,6 +132,39 @@ Cypress.Commands.add('addFlightWithFlightCode', (flightCode, estString, actStrin
   cy.request('POST', '/test/arrival', flightPayload);
 });
 
+Cypress.Commands.add('addFlightWithPaX', (flightCode, actPax, estString, actString, estChoxString, actChoxString, schString) => {
```

Comment:
Is the `X` in `addFlightWithPaX` deliberately in upper case?

Reply:
fixed